src/features/conversations/hooks/useCreateConversationAndSendMessage.hook.tsx

```
@@ -0,0 +1,55 @@
+import { Client } from "@twilio/conversations";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import { createChatConversation } from "./useCreateConversation.hook";
+import { formatError } from "utils/strings";
+import { ReactQueryKeys } from "constants/keys";
+import { sendMessage } from "./useSendMessage.hook";
+import { useTwilioClient } from "./useTwilioClient.hook";
+import { ConversationClientTypes } from "./useTwilioToken.hook";
+
+const createConversationAndSendMessage =
+  (twilioClient?: Client) =>
+  async ({
+    userIds,
+    name,
+    firstMessage,
+  }: {
+    userIds: string[];
+    name?: string;
+    firstMessage: string;
+  }) => {
+    try {
+      const createdConversation = await createChatConversation({
+        userIds,
+        name,
+      });
+      const conversationSid = createdConversation.conversation_sid;
+
+      const conversation = await twilioClient?.getConversationBySid(
+        conversationSid
+      );
+
+      const response = await sendMessage(conversation)(firstMessage).catch(
+        (e) => console.error("error here", formatError(e))
```

I noticed you have already wrapped this part in a try/catch. Any reason why you want to swallow the error here?

---

Ah thanks this was from debugging